src/hearings/containers/request-hearing/hearing-edit-summary/hearing-edit-summary.component.ts

```diff
@@ -323,6 +327,17 @@ export class HearingEditSummaryComponent extends RequestHearingPageFlow implemen
     return false;
   }
 
+  private pageVisitNonReasonalbleChangeExists(): boolean {
+    const caseFlagsModifiedDate = this.serviceHearingValuesModel.caseFlags.flags.map((flags) => flags.dateTimeModified);
```

Comment:
@DavidJayakumar - We have to also implement checks for case flags created date, as a flag can be created after the hearing request was created.